src/utils/encryption/t-encryption.test.tsx

```diff
@@ -1,28 +1,27 @@
 import { encrypt, decrypt } from '~utils';
 
 describe('CryptoJS Encrypt/Decrypt', () => {
-  const secretKey = 'key123';
+  beforeAll(() => {
+    process.env.VITE_CRYPTOJS_KEY = 'some-secret-key';
+  });
+
+  afterAll(() => {
+    delete process.env.VITE_CRYPTOJS_KEY;
+  });
```

Comment:
Instead of this, install dotenv in the setup.ts file and instantiate it... like this

import dotenv from 'dotenv';

dotenv.config();

that's all you need.